### DISCUSSION TITLE
How to disable autofocus on button element?

### DISCUSSION THREAD

**Discussion by maximyugov** [primefaces/community#1130] [PrimeVue] [answered]
Hi

My application is running in iframe and I'v got an error "Blocked autofocusing on a button element in a cross-origin subframe."
![blocked](https://github.com/primefaces/community/assets/78841025/b9821c9c-c859-48c6-b9e6-116945799947)

This error appears only in Chrome. But I need no errors in console. What can I do with that? How can I disable autofocusing to make this error disappear?

Update: error message appears when Toast message is shown. So I think PrimeVue tries to return focus on button, but fails.

Thanks

**Comment by tugcekucukoglu** [comment, marked as answer]
You could use `pt` property. For example:

```
  <Toast
            :pt="{
                closeButton: {
                    autofocus: false
                }
            }"
        />
```

**Reply by maximyugov** [reply]
Thank you
